src/groupeddataframe/complextransforms.jl

```diff
@@ -17,16 +17,18 @@ function _combine_multicol((firstres,)::Ref{Any}, wfun::Ref{Any}, gd::GroupedDat
         idx_agg = NOTHING_IDX_AGG
     end
     return _combine_with_first(Ref{Any}(wrap(firstres)), wfun, gd, wincols,
-                               Val(firstmulticol), idx_agg)
+                               Ref{Any}(Val(firstmulticol)), idx_agg)
 end
 
 function _combine_with_first((first,)::Ref{Any},
                              (f,)::Ref{Any}, gd::GroupedDataFrame,
                              (incols,)::Ref{Any},
-                             firstmulticol::Val, idx_agg::Vector{Int})
+                             (firstmulticol,)::Ref{Any}, idx_agg::Vector{Int})
+    @assert first isa Union{NamedTuple, DataFrameRow, AbstractDataFrame}
     @assert f isa Base.Callable
     @assert incols isa Union{Nothing, AbstractVector, Tuple, NamedTuple}
     @assert first isa Union{NamedTuple, DataFrameRow, AbstractDataFrame}
+    @assert firstmulticol isa Union{Val{true}, Val{false}}
```

Comment:
A despecialized `Val` is quite paradoxical. How about making this a `Bool`?

Reply:
good point - I will change it. The problem was that for `Val` compiler genterates methos for `Val{true}`, `Val{false}` and `Val{T} where T` and this last method is not needed by us.